src/mongoc/mongoc-stream.h

```diff
@@ -61,6 +61,7 @@ struct _mongoc_stream_t
    ssize_t          (*poll)            (mongoc_stream_poll_t *streams,
                                         size_t                nstreams,
                                         int32_t               timeout);
+   void             (*failed)          (mongoc_stream_t *stream);
```

Comment:
crap.  Yeah.  We want to make sure that sizeof(mongoc_stream_t) is consistent between 1.1 and 1.2